torchvision/prototype/datapoints/_bounding_box.py

```diff
@@ -175,8 +175,14 @@ def perspective(
         fill: FillTypeJIT = None,
         coefficients: Optional[List[float]] = None,
     ) -> BoundingBox:
+        # FIXME: why didn't this trigger CI???
         output = self._F.perspective_bounding_box(
-            self.as_subclass(torch.Tensor), startpoints, endpoints, self.format, coefficients=coefficients
+            self.as_subclass(torch.Tensor),
```

Comment:
IIUC this is a bugfix unrelated to the goal of the PR? (fine to keep it here, just want to make sure)